bin/coqtop.ml

```diff
@@ -39,18 +39,20 @@ let term =
         let context = Dune_rules.Super_context.context sctx in
         (* Try to compute a relative path if we got an absolute path. *)
         let coq_file_arg =
-          let cwd = Path.external_ Path.External.initial_cwd in
-          let file =
-            (* Best-effort symbolic link unfolding. *)
-            let file = Fpath.follow_symlinks coq_file_arg in
-            Option.value file ~default:coq_file_arg
-          in
-          let file = Path.of_filename_relative_to_initial_cwd file in
-          let cwd = Path.to_string cwd in
-          let file = Path.to_string file in
-          match String.drop_prefix ~prefix:(cwd ^ "/") file with
-          | None -> coq_file_arg
-          | Some s -> s
+          if Filename.is_relative coq_file_arg then coq_file_arg
+          else
+            let cwd = Path.external_ Path.External.initial_cwd in
```

Comment:
side note: maybe you should just reject absolute paths?  we don't allow `dune build /foo/bar` as far as I know. Why allow it here?

Reply:
Yeah, I know that's a bit silly. The reason I did that originally is that emacs/proof-general feeds `coqtop` with an absolute path. But I'm not sure how important that is that we support that.

Reply:
Should we also remove that @Alizter? I agree this is kind of useless anyway: editors will need to detect the workspace root anyway, so they can cook up a path relative to it.